packages/coding-standards/composer.json

```diff
@@ -11,7 +11,7 @@
         }
     ],
     "conflict": {
-        "symfony/dependency-injection": ">=4.4.19"
+        "symfony/dependency-injection": ">=4.4.19 <5.0.0 || 5.1.11 || >=5.2.2"
```

Comment:
What if symfony release version 5.1.12?

Reply:
According to [symfony release calendar](Symfony Releases Calendar) the 5.1 version reached the end of life so I expect no further patches.